packages/ffe-account-selector-react/src/index.d.ts

```diff
@@ -70,6 +70,7 @@ declare class AccountSelectorMulti<
 
 export interface BaseSelectorProps<T> {
     suggestions: Array<T>;
+    renderSuggestions: (suggestions: Array<T>) => void;
```

Comment:
Har det muligen snikit seg in ett `s` her. jag finner inget `renderSuggestions` deremot `renderSuggestion`. Slik jag tolker koden skall riktig type vare?
```
renderSuggestion: (suggestion: T) => React.ReactNode;
```